### PR TITLE
add files for IMT2020004

### PR DESCRIPTION
**Pull request by sooraj948**
Hey I am Sooraj Sathish. I hope this works. 

There is an extra file img/IMT2020004-background.jpg:Zone.Identifier . I am not sure what this is . Probably because the image was originally a png but I changed it to a jpg.

### CODE REVIEW

File: _posts/2021-01-01-IMT2020004.markdown [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+title:  "Sooraj Sathish"
+subtitle: ""
+author: "Sooraj"
+image: "img/IMT2020004-background.jpg"
+date:   2021-01-01 12:12:12
+---
+
+## About me
+
+"Hey I am Sooraj from Bangalore. I love watching anime and listening to podcasts while cycling. Hope the campus opens soon!!"
```
Comment (jaggu21) [inline _posts/2021-01-01-IMT2020004.markdown]: You don't need the quotes here :)

Comment (sooraj948) [reply _posts/2021-01-01-IMT2020004.markdown]: Do i change it and push it again?

**Comment by GaganAgarwal77** [comment]
Hi @sooraj948 , great work on the PR👍 , you don't need the extra file, while adding changes( git add) don't add that file, also Change your markdown file to include social media links to your facebook/instagram , I will close this Pull Request and you can send a fresh one with the changes 🙂, You can comment here if you have any doubts.

File: _posts/2021-01-01-IMT2020004.markdown (hunk 1 above)
Comment (kernelpanic77) [reply _posts/2021-01-01-IMT2020004.markdown]: Just make a commit on your forked repo with the change

**Comment by sooraj948** [comment]
I do not have any social media accounts :). Also removed  "s. Hope it works

**Review by kernelpanic77 — CHANGES_REQUESTED** [review]
Hey Sridhar!
Pls Remove this `img/IMT2020004-background.jpg:Zone.Identifier` file, it is empty and not required.

**Comment by GaganAgarwal77** [comment]
I have deleted that extra file myself @sooraj948 , next time remember to add only those files which you want to upload, well done once again 👍🏽

**Comment by sooraj948** [comment]
thanks bro @GaganAgarwal77 .

**Pull request merged**